### PR TITLE
Corrected a typo in File's documentation

### PR DESCRIPTION
**Pull request by Givralix**

### CODE REVIEW

File: src/file.cr [hunk 1]
```diff
@@ -568,7 +568,7 @@ class File < IO::FileDescriptor
     Crystal::System::File.link(old_path, new_path)
   end
 
-  # Creates a symbolic link at *new_path* to an existing file given by *old_path.
+  # Creates a symbolic link at *new_path* to an existing file given by *old_path*.
```
Comment (straight-shoota) [inline src/file.cr]: Does `old_path` really need to exist and be a file?

Comment (Givralix) [reply src/file.cr]: The function does let you create broken symlinks.

Comment (Sija) [reply src/file.cr]: More apt would be `src_path` + `dest_path`.

Comment (straight-shoota) [reply src/file.cr]: Or `target` and `link_name` from `man ln`. But changing the argument names should probably be a separate PR (unless @Givralix want's to tackle this here).

Comment (Givralix) [reply src/file.cr]: I think it's better if you do it.

**Pull request merged**